grails-app/views/consentGroupReview/_index.gsp

```diff
@@ -0,0 +1,16 @@
+<g:javascript>
+
+    const component = {
+      consentKey:'${issue.projectKey}',
+      consentGroupUrl: "${createLink(controller: 'consentGroup', action: 'getConsentGroup')}",
+      isAdmin: "${session.isAdmin}"
```

Comment:
I don't think this is a good place for non-constants. The concept behind `component` is that they are constants that will eventually be refactored into a common location available to all of the react code. Instead, this should be a service call. 

I have the same problem with `consentKey` but I don't have a good solution for this right now. We should reconsider this approach and see if there's a way to get this value in a different way.